patches/third_party-widevine-cdm-widevine.gni.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/third_party/widevine/cdm/widevine.gni b/third_party/widevine/cdm/widevine.gni
+index 58f073ca562ca40c3165f4dd0f650dc467b43d5c..735a99c71e5086e146ec0bdbf2090d56bd5f08ca 100644
+--- a/third_party/widevine/cdm/widevine.gni
++++ b/third_party/widevine/cdm/widevine.gni
+@@ -31,6 +31,7 @@ library_widevine_cdm_available =
+     (target_os == "mac" && (target_cpu == "x64" || target_cpu == "arm64")) ||
+     (target_os == "win" &&
+      (target_cpu == "x86" || target_cpu == "x64" || target_cpu == "arm64"))
++if (target_os == "linux") { library_widevine_cdm_available = true } # Let Linux users manually supply Widevine on architectures where it isn't publicly available.
```

Comment:
can you submit an upstream CL to put `library_widevine_cdm_available` in `declare_args`?

Reply:
I have started the process of getting `library_widevine_cdm_available` into `declare_args` in upstream but do not know whether or when it would land. Can we merge this PR as-is? I'll follow through on getting the work into upstream (and then hopefully to update our implementation to use the GN arg). But until then, we will have already delivered value to users.

Reply:
> I have started the process of getting `library_widevine_cdm_available` into `declare_args` in upstream but do not know whether or when it would land. Can we merge this PR as-is? I'll follow through on getting the work into upstream (and then hopefully to update our implementation to use the GN arg). But until then, we will have already delivered value to users.

if the CL is in the works, pls consider updating this PR to have a patch that adds `declare_args() { }` around `library_widevine_cdm_available` declaration and passing `library_widevine_cdm_available=true` as a gn arg if `targetOS == linux`.

Reply:
The CL is in the works in the following sense: I've never contributed to Chromium. I tried to create an issue on https://bugs.chromium.org/p/chromium/issues/wizard a few hours ago. The wizard ended with a message along the lines of "thanks, it will take some time for the issue to appear". I didn't receive a confirmation email or anything similar. I suspect the issue is in a review queue to prevent spam. I also have a local branch with the necessary changes. Then I believe I will need to `git cl upload` (and probably some other commands) to start the review process.

Reply:
@goodov I implemented your suggestion in 8f475ce95bfc61dd4f081d90790721859aff77f0. Wdyt?

Reply:
@mherrmann the idea is that it should be exactly the patch that gets submitted to upstream. That way it just goes away on its own when we update chromium. There should be no additional comments, etc...

Reply:
I've submitted the patch upstream at https://chromium-review.googlesource.com/c/chromium/src/+/5186136 and am hoping for their review.

Reply:
Upstream's initial response was to reject my patch:

> Widevine CDM isn't supported on Linux arm64 so I am afraid we cannot approve this cahnge.

I'm still in touch with them to try to convince them that this change makes sense.

Reply:
@bridiver I don't think I will manage to convince upstream to `put library_widevine_cdm_available` in `declare_args` so I undid my changes to that effect. Can we merge this PR as-is? If upstream do decide to accept the GN arg, then I will switch to the GN arg based implementation in a follow-up PR.